BRANCHES.md

```diff
@@ -28,14 +28,22 @@ the API of 3.(x+1) is backward compatible with 3.x). We only break API
 compatibility on major version changes (e.g. from 3.x to 4.0). We also maintain
 ABI compatibility within LTS branches; see the next section for details.
 
-## Backwards Compatibility
+## Backwards Compatibility for application code
```

Comment:
Minor: use of backward(s) in this and the following section is inconsistent, and there is variation in case used in the two headings.

Reply:
This is minor and preexisting so I don't intend to fix it here.